src/devices/GrovePi/Sensors/DigitalOutput.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.GrovePiDevice.Models;
+using System;
+using System.Collections.Generic;
+using System.Device.Gpio;
+
+namespace Iot.Device.GrovePiDevice.Sensors
+{
+    /// <summary>
+    /// DigitalOutput class is a generic output class
+    /// </summary>
+    public class DigitalOutput : ISensor<PinValue>
+    {
+        internal GrovePi _grovePi;
+        internal PinValue _value;
+
+        /// <summary>
+        /// DigitalOutput constructor
+        /// </summary>
+        /// <param name="grovePi">The GrovePi class</param>
+        /// <param name="port">The grove Port, need to be in the list of SupportedPorts</param>
+        public DigitalOutput(GrovePi grovePi, GrovePort port)
+        {
+            if (!SupportedPorts.Contains(port))
```

Comment:
The name of the argument should be included in the exception and you're not actually using the interpolated string.

``` C#
throw new ArgumentException($"Error: grove port {port} not supported.", nameof(port));
```

Reply:
if we are changing it then I'd get rid of the `Error: ` prefix since that is implied by throwing an exception